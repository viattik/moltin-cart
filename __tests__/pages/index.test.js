/* eslint-env jest */

import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import Index from 'pages/index.js';

describe('Index Page', () => {
  it('Index shows cart and list without data', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
