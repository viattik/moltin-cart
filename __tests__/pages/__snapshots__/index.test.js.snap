// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page Index shows cart and list without data 1`] = `
<div
  className="container-fluid"
>
  <div
    className="cart"
  >
    <div
      className="cart-content"
    >
      <table
        cellPadding="0"
        cellSpacing="0"
      >
        <thead>
          <tr>
            <th
              className="-item"
            >
              Item
            </th>
            <th
              className="-price"
            >
              Price
            </th>
            <th
              className="-qty"
            >
              Qty
            </th>
            <th
              className="-total"
            >
              Subtotal
            </th>
            <th
              className="-remove"
            />
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              colSpan={5}
            >
              No items in cart yet. Go get something!
            </td>
          </tr>
        </tbody>
        <tfoot>
          <tr>
            <td
              colSpan={3}
            >
              Total
            </td>
            <td
              className="-total"
              colSpan={2}
            >
              $0.00
            </td>
          </tr>
          <tr>
            <td
              className="-buttons"
              colSpan={5}
            >
              <div
                className="buttons-wrapper"
              >
                <a
                  className="icon-link clear-button"
                  href=""
                  onClick={[Function]}
                >
                  <i
                    className="icon-delete-button"
                  />
                  Clear cart
                </a>
                <a
                  className="icon-link checkout-button"
                  href=""
                  onClick={[Function]}
                >
                  <i
                    className="icon-transport"
                  />
                  Checkout
                </a>
              </div>
            </td>
          </tr>
        </tfoot>
      </table>
    </div>
    <div
      className="cart-sidebar"
      onClick={[Function]}
      tabIndex="0"
    >
      <i
        className="cart-sidebar-icon icon-package-carrier"
      />
      <div
        className="cart-sidebar-amount"
      >
        <span>
          0
        </span>
        items
      </div>
    </div>
    <div
      className="cart-close"
    >
      <button
        onClick={[Function]}
      >
        <i
          className="icon-multiplication-sign"
        />
      </button>
    </div>
  </div>
  <div
    className="page-index"
  >
    <ul
      className="row"
    />
  </div>
</div>
`;
